administrator/components/com_media/Controller/Api.php

```diff
@@ -104,7 +105,20 @@ public function files()
 			switch ($method)
 			{
 				case 'get':
-					$data = $this->getModel()->getFiles($adapter, $path, $this->input->getWord('filter'));
+					$option = $this->input->getWord('opt');
```

Comment:
No actually as we discussed we only need to get URL iff requested, so IMO we should not send it along file attributes at all, only when it requested we send URL along with request for selected file